lib/mix/tasks/appsignal.diagnose.ex

```diff
@@ -86,10 +94,107 @@ defmodule Mix.Tasks.Appsignal.Diagnose do
     IO.puts("  Language: Elixir")
     IO.puts("  Package version: #{library_report[:package_version]}")
     IO.puts("  Agent version: #{library_report[:agent_version]}")
-    IO.puts("  Agent architecture: #{library_report[:agent_architecture]}")
     IO.puts("  Nif loaded: #{yes_or_no(library_report[:extension_loaded])}")
   end
 
+  defp fetch_installation_report do
+    download_report =
+      case do_fetch_installation_report("download") do
```

Comment:
Does this match both the success and error case?

Reply:
Yes, the `parsing_error` scenario is the error case. I'll see if I can wrap it in an `ok`|`error` tuple